vpc/service/long_lived_lock.go

```diff
@@ -19,6 +24,81 @@ const (
 	timeBetweenTryingToAcquireLocks = 15 * time.Second
 )
 
+func (vpcService *vpcService) readLocks(rows *sql.Rows) ([]*vpcapi.Lock, error) {
```

Comment:
I don't think there's any reason for this function to have a pointer receiver?

Reply:
No, not technically. However, it is [generally best practice ](https://golang.org/doc/faq#methods_on_values_or_pointers) to use the same receiver type, e.g. if one method is a pointer receiver, all should be. This is so that the [method set](https://golang.org/doc/faq#different_method_sets) is consistent.